corehq/apps/domain/templates/domain/select.html

```diff
@@ -10,34 +10,84 @@
   {% initial_page_data 'invitation_links' invitation_links %}
   {% initial_page_data 'domain_links' domain_links %}
 
-  <div id="all-links" class="ko-template">
-    <div class="row">
-      <div class="col-sm-4">
-        <ul class="list-unstyled" data-bind="visible: invitationLinks().length, foreach: invitationLinks">
-          <li style="padding-bottom: 6px;">
-            <a data-bind="attr: {href: url}" class="btn btn-primary btn-xs">
-              <i class='fa fa-envelope'></i>
-              {% trans "Accept" %}
-            </a>
-            <!-- ko text: display_name --><!-- /ko -->
-          </li>
-        </ul>
-        <ul class="nav nav-pills nav-stacked" data-bind="foreach: domainLinks">
-          <li><a data-bind="attr: {href: url}, text: display_name"></a></li>
-        </ul>
-      </div>
-      <div class="col-sm-4">
-        <!-- spacer -->
-      </div>
-      <div class="col-sm-4">
-        <div class="pull-right">
+<div id="all-links" class="ko-template">
+  <div class="btn-toolbar" data-bind="visible: invitationLinks().length">
+    <a class="btn btn-primary"
+       href="{% url "accept_all_invitations" %}">
+      <i class="fa fa-envelope"></i>
+      {% trans 'Accept All Invitations' %}
+    </a>
+  </div>
+  <p/>
+
+  <div class="panel panel-info" data-bind="visible: invitationLinks().length">
+    <div class="panel-heading ">
+      <h3 class="panel-title" style="padding-top: 7px;">
```

Comment:
You don't need the inline style here. The "My Projects" header uses it to vertically center the text because the search box there makes the panel header taller.